app/entities/user.go

```diff
@@ -85,3 +85,8 @@ func checkPass(password string) error {
 
 	return nil
 }
+
+// Hide sencitive data while printing
+func (u User) String() string {
```

Comment:
is it normal to use name "String"  ?

Reply:
It is normal, because I write custom stringer function and it must have the signature of stringer `String() string`